Modules/RosterManager/Options.lua

```diff
@@ -448,9 +443,31 @@ local function dynamic_item_values(roster, equationGet, equationSet, multiplierG
         set = (function(i, v) multiplierSet(tonumber(v)) end),
         name = CLM.L["Multiplier"],
         pattern = CONSTANTS.REGEXP_FLOAT,
+    }
+	order = order + 1
+    args["equation_expvar"] = {
+        type = "input",
+        desc = CLM.L["Exponential scaling value used by the equations; Base for EPGPWeb, or Exponent for WoWpedia"],
+        order = order,
+        width = 0.5,
+        get = (function(i) return tostring(expvarGet()) end),
+        set = (function(i, v) expvarSet(tonumber(v)) end),
+        name = CLM.L["ExpVar"],
```

Comment:
I do think this  name is not enough descriptive for user maybe:
```suggestion
        name = CLM.L["Exponent / Base"],
```